### DISCUSSION TITLE
Is it possible to get current entity instance inside a custom type?

### DISCUSSION THREAD

**Discussion by suerta-git** [mikro-orm/mikro-orm#2826] [Q&A]
Hi there,

I know it's a little weird, but I'd like to know if it's possible to make my custom type to know in which entity it is used.

If it's impossible, I think getter/setter type property may also can solve my problem, are there some examples for me to refer to?

**Comment by B4nan** [comment]
Nope, custom types are stateless and not bound to any entity, single instance of one type is used for all occurrences. 

What is the use case? Maybe hooks could help?

**Reply by suerta-git** [reply]
Ah thanks, I think I can adjust my workflow and pass the instance to resolved property.

If I have more other questions I will let you know.
